frontend/src/components/CoronaActivitiesComponents/ProvinceCapitalWeather.js

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components/macro";
+import {useEffect, useState} from "react";
+import {getWeatherDetails} from "../../service/weatherApiService";
+
+export default function ProvinceCapitalWeather({province}) {
+
+    const [weatherDetails, setWeatherDetails] = useState()
+
+    useEffect(() => {
+        getWeatherDetails(province).then(setWeatherDetails)
+    }, [province])
+
+    return (
+        <WeatherWrapper>
+            {weatherDetails &&
+            <>
+                <p>{weatherDetails.capital} ({weatherDetails.weatherState})</p>
+                <img src={weatherDetails.weatherIconUrl} alt={weatherDetails.weatherState}/>
+                <MinMaxTemp>
+                    <p>Min: {weatherDetails.minTemperature}°</p>
+                    <p>Max: {weatherDetails.maxTemperature}°</p>
+                </MinMaxTemp>
+            </>
+            }
+        </WeatherWrapper>
+    )
```

Comment:
What do you think about this? Imho its much easier to read
```suggestion

    if(!weatherDetails){
        return null
    }

    return (
        <WeatherWrapper>
                <p>{weatherDetails.capital} ({weatherDetails.weatherState})</p>
                <img src={weatherDetails.weatherIconUrl} alt={weatherDetails.weatherState}/>
                <MinMaxTemp>
                    <p>Min: {weatherDetails.minTemperature}°</p>
                    <p>Max: {weatherDetails.maxTemperature}°</p>
                </MinMaxTemp>
        </WeatherWrapper>
    )
```

Reply:
would it be `return null` or `return <></>` ?